draft/spec/index.html

```diff
@@ -868,6 +868,28 @@ <h2>Extensions</h2>
             </p>
         </blockquote>
     </section>
+    
+    <section id="filesystem-features">
+        <h2>Filesystem features</h2>
+        <p>
+            In order to maximize the compatibility of the OCFL with different filesystems, and thus improve the
+            portability of OCFL Objects between different systems, some restrictions on the use of certain
+            filesystem features are necessary. If the preservation of non-OCFL-compliant features is required then
+            data MUST be wrapped in a suitable disk or filesystem image format which OCFL can treat as a regular
+            file.
+        </p>
+        <ol>
+          <li>Filesystem metadata (e.g. permissions, access, and creation times) are not considered portable between
+              filesystems or preservable through file transfer operations. Nor can these attributes be validated in
+              terms of fixity in a consistent manner. As such, the OCFL does not support the portability of these
+              attributes.</li>
+          <li>Hard and soft (symbolic) links are not portable and MUST NOT be used within OCFL Storage hierachies.
+              A common use case for links is storage deduplication. OCFL inventories provide a portable method
+              of achieving the same effect.</li>
+          <li>Transparent filesystem features such as compression and encryption should be effectively invisible to
+              OCFL operations. Consequently, they should not be expected to be portable.</li>
```

Comment:
To what effect?

Reply:
@ahankinson - are you suggesting we remove this bullet entirely?

Reply:
No; I'm saying that we should expand a bit on what that could mean. Moving from a compressed to an uncompressed filesystem may significantly increase the size of your data. Likewise, simply cloning an encrypted disk may make it unreadable. Simply saying they are not portable hides more important issues IMO.

Reply:
@ahankinson do you have language to suggest?  I'm struggling with finding succinct sentences.